socialhome/streams/app/components/AuthorBar.vue

```diff
@@ -44,48 +40,74 @@
 </template>
 
 <script>
-import Vue from "vue"
```

Comment:
@jaywink: Please keep `<script>` content indented. `<template>` one is. It feels weird that `<script>` is not.